src/main/groovy/com/ullink/AssemblyInfoVersionPatcher.groovy

```diff
@@ -13,7 +13,7 @@ class AssemblyInfoVersionPatcher extends ConventionTask {
     List<String> projects = []
 
     AssemblyInfoVersionPatcher() {
-        conventionMapping.map "projects", { [ project.tasks.msbuild.mainProject?.projectName ] }
+        conventionMapping.map "projects", { project.tasks.msbuild.projects.collect { it.name } }
```

Comment:
that'll patch all projects in solution right ? not sure we want to patch assembly version tokens for test or ref projects, do we ?